shared.go

```diff
@@ -349,6 +350,21 @@ func (parsed rawHeader) getP2S() (*byteBuffer, error) {
 	return parsed.getByteBuffer(headerP2S)
 }
 
+// getB64 extracts parsed "b64" from the raw JSON, defaulting to true.
+func (parsed rawHeader) getB64() (bool, error) {
```

Comment:
Would it ever be necessary here to distinguish between the header being present, and set, or the header being not present and the default return value being false?